benchmarks/src/index.ts

```diff
@@ -74,19 +76,26 @@ function writeOutput({
       width: 1280,
       height: 960
     });
-    // page.on('console', (msg: playwright.ConsoleMessage) => {
-    //   console.log(`browser type=${msg.type()} text=${msg.text()}`);
-    // });
+    /*
+    page.on('console', (msg: playwright.ConsoleMessage) => {
+      console.log(`browser type=${msg.type()} text=${msg.text()}`);
+    });
+    */
     /**
      * Wait for a widget to be visible.
      */
     async function waitForNotebook(id: string): Promise<void> {
-      await page.waitForSelector(`#${id}`, { visibility: 'visible' });
```

Comment:
Ha, I was just working on this exact same change! I moved it over to a global config, instead of passing it in function, what do you think of that?

```
    context.setDefaultTimeout(TIME_OUT);
```

Reply:
Nevermind, it's also fine as is, can always move around later.

Reply:
Obviously I have been searching on the playwright API but have not found it directly. Will pull that in this PR as much better.

Reply:
pushed too late :) can you pull that default timeout in https://github.com/jupyterlab/jupyterlab/pull/8786? Thx